lib/features/modeling/cmd/AlignElementsHandler.js

```diff
@@ -25,22 +25,22 @@ AlignElements.prototype.preExecute = function(context) {
       y: 0
     };
 
-    if (alignment.left) {
+    if (isNumber(alignment.left)) {
```

Comment:
Can `alignment[type]` be string? Cause Javascript also support `'1' + 1 = 2`.

Reply:
Let us assume it cannot. We could guard against this by using `isDefined(alignment.left)` instead of the number check.